scripts/fmt.sh

```diff
@@ -16,6 +16,11 @@
 pre-commit run --all-files
 RETURN_CODE=$?
 
+# cat this file for helping on identifying the root cause when some issue happens
+if [ -f /tmp/pre-commit.log ]; then
+  cat /tmp/pre-commit.log
```

Comment:
I would prefer to create the file in the project (preferable "hidden") `$(pwd)/.pre-commit.log` and also add it to the `.gitignore` and `.dockerignore` files. This way, when working from the IDE you don't have to traverse directories outside of the project in order to see the errors.